code/game/gamemodes/autotraitor/autotraitor.dm

```diff
@@ -96,7 +96,8 @@
 			for(var/obj/item/implant/mindshield/I in H.contents)
 				if(I && I.implanted)
 					possible_traitors -= player
-
+			if(isgolem(H))
```

Comment:
if you remove the bits that set `job` on the silencium and bananaium golems, you can instead do a check on `H.job` here, which should work for any non-crew humanoid instead of specifically just golems

Reply:
Unfortunately clown/mime reagent healing is checked on that var
I will take look into the best way to do this